specification/web/resource-manager/readme.md

```diff
@@ -34,7 +34,7 @@ These are the global settings for the Web API.
 title: WebSiteManagementClient
 description: WebSite Management Client
 openapi-type: arm
-tag: package-2019-08
+tag: package-2020-06
```

Comment:
For task: Making 2020-06-01 the default api-version (https://msazure.visualstudio.com/Antares/_sprints/taskboard/Domains%20and%20Certificates/Antares/UX/s150?workitem=8469099). @naveedaz are there any other changes needed for this?